README.md

```diff
@@ -91,7 +91,7 @@ To run the project, execute the following command:
 - LinkedIn: [roland-ntwali](https://www.linkedin.com/in/roland-ntwali/)
 
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be great if you make your commit messages capitalize to make your pull request professional and descriptive 👍
- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍